CMSgov/easi-app
# EASI-2508 Edits requested experience (Requester)

# EASI-2508

## Description
A warning to "View feedback" will show up in the Trb Request Form when `request.taskStatuses.feedbackStatus` is `TRBFeedbackStatus.EDITS_REQUESTED`

## Changes
- New `TechnicalAssistance/RequestForm/Feedback` view
  - Shows a list of feedback messages
- New `EditsRequestedWarning` alert component 
  - Modify `StepHeader` so that it has a property for the warning component
- Refactor `formStepSlugs` and `viewSlugs` to clear up url view handling
  - New types `FormStepSlug` and `ViewSlug`

## Notes
- The feedback list is reversed on the frontend. Not sure if there's an option from the backend to reverse the order.
- The list is also filtered by `TRBFeedbackAction.REQUEST_EDITS`

## Testing
- Jest test `TechnicalAssistance/RequestForm/index.test.tsx`: `TRB Request Form Feedback`
- See https://github.com/CMSgov/easi-app/pull/1824 on how to create Feedback messages with `createTRBRequestFeedback`
  - Make sure to use Eua Ids that are available from [mock data](https://github.com/CMSgov/easi-app/blob/f805ded627e337d8db1094e66665bc38dd63db7b/pkg/local/cedar_ldap.go)
